workers/data_refinery_workers/downloaders/management/commands/queue_downloader.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
```

Comment:
Probably better to put it at the very top so that a reader can find it immediately. And the usage part will fit better at the top too because it is the usage of this management command (instead of the class `Command`).